modules/ROOT/pages/pricing-metrics.adoc

```diff
@@ -52,6 +52,10 @@ Anypoint Platform captures usage metrics across products, but not all metrics ar
 |Data Graph	
 |_DataGraph orchestration_: An API request made by Anypoint DataGraph to the source APIs to get data for the GraphQL API request made to Anypoint DataGraph. 
 |Orchestrations are not currently aggregated on usage reports.
+
+|IDP 
+|_Intelligent Document Processing (IDP) Document Pages_: A single page processed by IDP.
+|IDP document actions may process documents that have more than one page with each page counting separately. Executing document actions from RPA also counts Document Pages and additionally consumes the corresponding RPA Bot Minutes accounting for the time the RPA process runs. 
```

Comment:
```suggestion
|IDP document actions might process documents that have more than one page, with each page counting separately. Executing document actions from RPA also counts Document Pages and additionally consumes the corresponding RPA Bot Minutes accounting for the time the RPA process runs. 
```

Reply:
Is the capitalization correct for Document Pages? I know we typically don't capitalize the general use of a term, but this might be an exception.

The wording on this one is a bit hard to follow. I've made some suggestions, but of course, I leave it to you to determine what's best.